setup.py

```diff
@@ -56,6 +56,13 @@
           'schema-salad >= 2.6.20170927145003, < 3',
           'typing >= 3.5.3',
           'six >= 1.8.0',
+          'prov == 1.5.1',
+          'pathlib == 1.0.1',
+          'graphviz == 0.8.2',
+          'uuid == 1.30',
```

Comment:
hmm.. this library might not be Python 3 compatible ..

Reply:
Yeah, not updated since 2007 :-P https://pypi.python.org/pypi/uuid

Reply:
It is a backport of the functionality within the Python 2.5+ standard library, so remove `uuid` from setup.py and requirements.txt and I bet many more of the tests will pass

Reply:
https://docs.python.org/2/library/uuid.html

Reply:
Can we still use it even if removed from setup.py and requirement.txt?

Reply:
Yes, uuid is part of the Python Standard Library from Python 2.5 and onwards: https://docs.python.org/2/library/uuid.html